### DISCUSSION TITLE
Add SUSE font to next/font/google

### DISCUSSION THREAD

**Discussion by solkkku** [vercel/next.js#69963] [Ideas]
### Goals

1. Being able to use SUSE with next/font/google

### Background

I wanted to use SUSE in one of my projects and it isn't available in next/font/google, at least in 14.2.8. I would appreciate it if it was added. 

https://fonts.google.com/specimen/SUSE

### Proposal

1. Add SUSE to next/font/google

**Comment by mrtanatorn44** [comment]
Try local fonts
https://nextjs.org/docs/app/building-your-application/optimizing/fonts#local-fonts
